kafka/kafka.sh

```diff
@@ -168,8 +174,25 @@ function install_and_configure_kafka_server() {
   wait_for_kafka
 }
 
+function install_kafka_python_package() {
+  KAFKA_PYTHON_PACKAGE="kafka-python==2.0.2"
+  if [[ "$(echo "${DATAPROC_IMAGE} > 2.0" | bc)" -eq 1 ]]; then
+    /opt/conda/default/bin/pip install "${KAFKA_PYTHON_PACKAGE}" || { sleep 10; /opt/conda/default/bin/pip install "${KAFKA_PYTHON_PACKAGE}"; }
+  else
+    OS=$(. /etc/os-release && echo "${ID}")
+    if [[ "${OS}" == "rocky" ]]; then
+      yum install -y python2-pip
+    else
+      apt-get install -y python-pip
+    fi
+    pip2 install "${KAFKA_PYTHON_PACKAGE}" || { sleep 10; pip2 install "${KAFKA_PYTHON_PACKAGE}"; } || { sleep 10; pip install "${KAFKA_PYTHON_PACKAGE}"; }
+  fi
+}
+
 function main() {
+  recv_keys || err 'Unable to receive keys.'
   update_apt_get || err 'Unable to update packages lists.'
+  install_kafka_python_package
```

Comment:
This new functionality should be guarded by a new meta data, e.g., install-kafka-python with default false.

Reply:
Understood.